sdks/python/apache_beam/transforms/core.py

```diff
@@ -1448,18 +1448,6 @@ def get_output_type(self):
     return (self.get_type_hints().simple_output_type(self.label) or
             self.infer_output_type(None))
 
-  def expand(self, pbegin):
-    from apache_beam.io import iobase
-    assert isinstance(pbegin, pvalue.PBegin)
-    self.pipeline = pbegin.pipeline
-    coder = typecoders.registry.get_coder(self.get_output_type())
-    source = self._create_source_from_iterable(self.value, coder)
-    return (pbegin.pipeline
```

Comment:
<!--new_thread; commit:a6f460fe3b760aafbc748ae18956f0f2c1fedfad; resolved:0-->
*can* **we...**